src/main/java/no/ndla/taxonomy/rest/v1/dtos/nodes/searchapi/TaxonomyContextDTO.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Part of NDLA taxonomy-api
+ * Copyright (C) 2023 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.taxonomy.rest.v1.dtos.nodes.searchapi;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.net.URI;
+import java.util.*;
+
+public record TaxonomyContextDTO(
+        @JsonProperty @Schema(description = "The public-id of the node connected via content-uri") URI id,
+        @JsonProperty @Schema(description = "The id of the root parent of the context") URI subjectId,
```

Comment:
Kunne kanskje kalla denne rootId for å passe med dok.
```suggestion
        @JsonProperty @Schema(description = "The id of the root parent of the context") URI rootId,
```

Reply:
Den finnes allerede i [denne](https://github.com/NDLANO/backend/blob/master/search-api/src/main/scala/no/ndla/searchapi/model/search/SearchableTaxonomyContext.scala), så hadde vært nice å bare beholdt de samme navnene enn så lenge.

Reply:
Greit nok!